vowpalwabbit/boosting.cc

```diff
@@ -391,12 +393,13 @@ VW::LEARNER::base_learner* boosting_setup(VW::setup_base_i& stack_builder)
   // "adaptive" implements AdaBoost.OL (Algorithm 2 in BLK'15,
   // 	    using sampling rather than importance weighting)
 
-  logger::errlog_info("Number of weak learners = {}", data->N);
-  logger::errlog_info("Gamma = {}", data->gamma);
+  all.logger.err_info("Number of weak learners = {}", data->N);
+  all.logger.err_info("Gamma = {}", data->gamma);
 
   data->C = std::vector<std::vector<int64_t> >(data->N, std::vector<int64_t>(data->N, -1));
   data->t = 0;
   data->all = &all;
+  data->logger = all.logger;
```

Comment:
didn't we already copy the logger into `data` at the constructor line 374?

Reply:
You're right. Thanks!